awscli/paramfile.py

```diff
@@ -37,6 +37,10 @@
     'cloudformation.update-stack.stack-policy-url',
     'cloudformation.set-stack-policy.stack-policy-url',
 
+    'custom.cp.website-redirect',
```

Comment:
My preference is to just put in the long term fix rather than a temporary fix that will be removed once the changes are plumbed through clidriver/handlers.

However, I'm ok with putting this fix in for the time being.  Code and test themselves look good.  Feel free to merge if you want.